File: transfer-backend/src/main/java/br/com/transfer/service/ContaBancariaService.java
package br.com.transfer.service;

import java.util.List;

import br.com.transfer.entity.ContaBancaria;

public interface ContaBancariaService {
	
	ContaBancaria createContaBancaria(Integer usuarioId, ContaBancaria contaBancaria);

    ContaBancaria updateContaBancaria(Integer contaBancariaId, ContaBancaria contaBancaria);

    void deleteContaBancaria(Integer contaBancariaId);

    List<ContaBancaria> getContasBancarias();

    ContaBancaria getContaBancariaById(Integer contaBancariaId);

}

File: transfer-frontend/src/services/ContaBancariaService.js
import axios from "axios"

const CONTAS_BANCARIAS_ENDPOINT = "http://localhost:8080/api/contas_bancarias/"

class ContaBancariaService {
    createContaBancaria(usuarioId, contaBancaria){
        return axios.post(CONTAS_BANCARIAS_ENDPOINT + usuarioId, contaBancaria);
    }
    updateContaBancaria(contaBancariaId, contaBancaria){
        return axios.put(CONTAS_BANCARIAS_ENDPOINT + contaBancariaId, contaBancaria);
    }
    deleteContaBancaria(contaBancariaId){
        return axios.delete(CONTAS_BANCARIAS_ENDPOINT + contaBancariaId);
    }
    getContasBancarias(){
        return axios.get(CONTAS_BANCARIAS_ENDPOINT);
    }
    getContaBancariaById(contaBancariaId){
        return axios.get(CONTAS_BANCARIAS_ENDPOINT + contaBancariaId);
    }
}

export default new ContaBancariaService()
